faq/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: faq
+title: FAQs
+---
+
+## OBO Foundry Frequently Asked Questions (FAQs)
```

Comment:
this is taken care of by the `title` in the front matter and is therefore redundant

Reply:
thanks, fixed